serverless/README.md

```diff
@@ -14,12 +14,17 @@ aws cloudformation create-stack \
   --capabilities CAPABILITY_AUTO_EXPAND CAPABILITY_IAM
 ```
 
-If you are updating the macro after a new release, use the `update-stack` method instead with the same paramters:
+If you are updating the macro after a new release, create and execute a change set with the updated template:
 ```bash
-aws cloudformation update-stack \
+aws cloudformation create-change-set \
   --stack-name datadog-serverless-macro \
   --template-url https://datadog-cloudformation-template.s3.amazonaws.com/aws/serverless-macro/latest.yml \
-  --capabilities CAPABILITY_AUTO_EXPAND CAPABILITY_IAM
+  --capabilities CAPABILITY_AUTO_EXPAND CAPABILITY_IAM \
+  --change-set-name datadog-serverless-macro-<newest release version>
+
+aws cloudformation execute-change-set \
+  --stack-name datadog-serverless-macro \
+  --change-set-name datadog-serverless-macro-<newest release version>
```

Comment:
Ideally, we would want something like `aws cloudformation deploy` which does all of this in one step. However, the `deploy` command requires a template file instead of a template URL. Since we provide the updated `latest.yml` file through the s3 link, customers would have to download this file first before using the `deploy` command. With this solution, they can directly use the s3 link as the template URL.

On the other hand, this solution requires two more commands to update the stack, and it requires the change set to be unique each time, so they would have to add on the `<newest release version>` themselves and would have to look that up. (This part doesn't really have to be the newest release version, I just thought this would be an appropriate way for them to name change sets so they don't have duplicates, which results in an error.)